src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -585,6 +670,157 @@ private void rotateRightClicked()
     }
 
     /**
+     * Rotate the structure counter clockwise.
+     */
+    private void rotateLeftClicked()
+    {
+        rotation = (rotation + ROTATE_LEFT) % POSSIBLE_ROTATIONS;
+        updateRotation(rotation);
+    }
+
+    /**
+     * Rotate the structure counter clockwise.
+     */
+    private static void mirror()
+    {
+        Settings.instance.mirror();
+    }
+
+
+    /*
+     * Miscellaneous
```

Comment:
Id make it more like "------------- Miescellaneous ------------"
So its better to see